symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -48,6 +50,16 @@ void test_cwrapper() {
     printf("Is_a_Rational %s: %d\n", s, is_a_Rational(e));
     printf("Is_a_Integer %s: %d\n", s, is_a_Integer(e));
 
+    rational_set_ui(f, 76, 59);
```

Comment:
@shivamvats You can make a different function to test the wrappers for complex, and use `SYMENGINE_C_ASSERT` to check if they are correct rather than `printf`.